libsignal-service-dotnet/websocket/SignalWebSocketConnection.cs

```diff
@@ -138,11 +123,13 @@ public void SendResponse(WebSocketResponseMessage response)
                 Type = WebSocketMessage.Types.Type.Response,
                 Response = response
             };
-            WebSocket.OutgoingQueue.Add(message.ToByteArray());
+            SignalWebSocket.SendMessage(message.ToByteArray());
         }
 
         private void SendKeepAlive(CancellationToken token, object state)
         {
+            throw new NotImplementedException();

```

Comment:
Does this ever get called?

Reply:
Not at the moment, but apparently libsignal-service-java sends these. Might want to send them too in the future?